src/doing/open/commands.py

```diff
@@ -94,16 +94,17 @@ def pipe():
 
 
 @open.command()
-@click.argument("work_item_id")
+@click.argument("work_item_id", default=-1)
 def issue(work_item_id):
     """
-    Open a specific WORK_ITEM_ID.
+    Open a specific WORK_ITEM_ID or when not provided open the workitem based on the branchname.
```

Comment:
First sentence needs to be short (it's what displays when you type `doing open`)

So f.e.:

```
Open a specific WORK_ITEM_ID.

When not provided attempt to auto-detect the WORK_ITEM_ID using the git branch name
```

Reply:
rephrased